src/modals/NewAddressModal.tsx

```diff
@@ -20,18 +20,18 @@ import { AddressAndKeys, addressToGroup, deriveNewAddressData, TOTAL_NUMBER_OF_G
 import { Info } from 'lucide-react'
 import { useCallback, useEffect, useRef, useState } from 'react'
 
-import ExpandableSection from '../../components/ExpandableSection'
-import InfoBox from '../../components/InfoBox'
-import ColoredLabelInput from '../../components/Inputs/ColoredLabelInput'
-import KeyValueInput from '../../components/Inputs/InlineLabelValueInput'
-import Select from '../../components/Inputs/Select'
-import Toggle from '../../components/Inputs/Toggle'
-import Modal, { ModalFooterButton, ModalFooterButtons } from '../../components/Modal'
-import HorizontalDivider from '../../components/PageComponents/HorizontalDivider'
-import { Section } from '../../components/PageComponents/PageContainers'
-import { Address, useAddressesContext } from '../../contexts/addresses'
-import { useGlobalContext } from '../../contexts/global'
-import { getRandomLabelColor } from '../../utils/colors'
+import ExpandableSection from '../components/ExpandableSection'
+import InfoBox from '../components/InfoBox'
+import ColoredLabelInput from '../components/Inputs/ColoredLabelInput'
+import KeyValueInput from '../components/Inputs/InlineLabelValueInput'
+import Select from '../components/Inputs/Select'
+import Toggle from '../components/Inputs/Toggle'
+import ModalCentered, { ModalFooterButton, ModalFooterButtons } from '../components/ModalCentered'
```

Comment:
And maybe rename `Modal` into `ModalContainer` to be 100% explicit?

Reply:
We could, but since we don't rly have the need for another `SideModal`, I'd rather leave it as it is. I can do it if you insist.

Reply:
The `components` folder start to be quite large - I know - and I'd be in favour of starting to regrouping in theme, like I did with `inputs`.
But I made this remark because I was looking for the "side" modal variant (as I saw the "center" version) and couldn't find it... 
I can definitely see other cases where the side modal would be used to display complementary informations :)

Reply:
ok! Convienced! And I'll move it into the `modals` folder, together with all its other modal friends 🧑‍🤝‍🧑

Reply:
done